scripts/release/rpm/mariner.dockerfile

```diff
@@ -4,7 +4,8 @@ FROM ${image} AS build-env
 ARG cli_version=dev
 
 RUN tdnf update -y
-RUN tdnf install -y binutils file rpm-build gcc libffi-devel python3-devel openssl-devel make diffutils patch dos2unix perl sed
+# kernel-headers, glibc-devel, binutils are needed to install psutil python package on ARM64
```

Comment:
Any doc for this?

Reply:
In [psutil doc](https://github.com/giampaolo/psutil/blob/master/INSTALL.rst), only `gcc` and `python3-devel` are needed.
However, without `glibc-devel`, I get
```
/usr/lib/gcc/aarch64-unknown-linux-gnu/11.2.0/include-fixed/limits.h:203:15: fatal error: limits.h: No such file or directory
        203 | #include_next <limits.h>
``` 
Without `kernel-headers`, I get
```
/usr/include/bits/local_lim.h:38:10: fatal error: linux/limits.h: No such file or directory
         38 | #include <linux/limits.h>
            |          ^~~~~~~~~~~~~~~~
```
Without `binutils`, I get
```
aarch64-unknown-linux-gnu-gcc: fatal error: cannot execute 'as': execvp: No such file or directory
      compilation terminated.
```